### PR TITLE
Add testimonial to backup connect screen

### PR DESCRIPTION
**Pull request by CodeyGuyDylan**
Add testimonial to backup connect screen

## Proposed changes:

* Adds new testimonials component to js-packages/components
* Uses new component on backup connect screen

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

P2: p1HpG7-kd7-p2
Figma: bSsQjyTPK8VIfopwI4fdIU-fi-3005_2665

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

1. Checkout this branch locally or with the Jetpack Beta plugin
2. DONT connect Jetpack to your wordpress account
3. Install the Jetpack VaultPress Backup plugin
4. If you are not redirected there, go to `/wp-admin/admin.php?page=jetpack-backup`
5. You should see the new testimonial underneath the first module
image

6. Make sure it looks good on Mobile as well
image
image

7. Try out the left and right arrows on the module to make sure it goes through the quotes (right now there are only 2) 
image

### CODE REVIEW

File: projects/packages/backup/src/js/hooks/useConnection.js [hunk 1]
```diff
@@ -1,19 +1,59 @@
...
+	{
+		quote: __(
+			'Millions of people depend on my site, and downtime isn’t an option. Jetpack VaultPress Backup handles my site security and backups so I can focus on creation.',
+			'jetpack-backup-pkg'
+		),
+		author: 'Tim Ferriss',
+		profession: createInterpolateElement(
+			__( '<a>Author / Investor / Podcaster</a>', 'jetpack-backup-pkg' ),
+			{
+				a: <ExternalLink href="https://tim.blog/" target="_blank" rel="noreferrer noopener" />,
```
Comment (elliottprogrammer) [inline projects/packages/backup/src/js/hooks/useConnection.js]: Are we sure we want to be linking to external sources here?  And in the other slide also?

**Review by elliottprogrammer — DISMISSED** [review]
Tested and looks good and works good. Code looks good, and I don't see any issues.
Not a blocker, but would be nice if the testimonials had some sort of transition effect when clicking the arrows, not sure if that is expected or not.  Also left a comment about the external links.
Otherwise, this LGTM! 👍

File: projects/packages/backup/src/js/hooks/useConnection.js (hunk 1 above)
Comment (CodeyGuyDylan) [reply projects/packages/backup/src/js/hooks/useConnection.js]: On the Landing pages we link to these sources on jetpack.com/backup and jetpack.com/for/agencies, so I assumed we'd do that here too, but I am not sure. I can check with Simon

Comment (CodeyGuyDylan) [reply projects/packages/backup/src/js/hooks/useConnection.js]: Simon said to remove them: p1HpG7-kd7-p2#comment-63892 , so they are gone now and it is ready for re-review 😄

**Review by elliottprogrammer — DISMISSED** [review]
Looks great! Code looks all good!
LGTM! 👍

**Review by elliottprogrammer — APPROVED** [review]

**Pull request merged**